data/en/isdateobject.json

```diff
@@ -0,0 +1,15 @@
+{
+	"name":"isDateObject",
+	"type":"function",
+	"syntax":"isDateObject(value)",
+	"returns":"boolean",
+	"related":["isDate","isObject"],
+	"description":"Checks if the given value is a date / time object. The difference between isDateObject and isDate, latter returns true for date / time objects as well as date / time strings.",
+	"params": [
+		{"name":"value","description":"A value, usually the name of a variable","required":true,"default":"","type":"variableName","values":[]}
+	],
+	"engines": {
+		"coldfusion": {"minimum_version":"2016", "notes":"", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-functions/functions-in-k/isdateobject.html"}
```

Comment:
Based on the History section in Adobe's documentation, it seems like the minimum version is actually `2016.0.3`.

Reply:
Thanks for the comments. I will make changes when I find some time.

Reply:
Made the change and committed